src/Bicep.Core.UnitTests/Parsing/LexerTests.cs

```diff
@@ -103,6 +102,152 @@ public void UnterminatedMultiLineComment_ShouldBeRecognizedWithError()
                 expectedStartPosition: expectedTokenText.Length);
         }
 
+        [TestMethod]
+        public void MissingCodesInDisableNextLineDiagnosticsDirective_ShouldBeRecognizedWithError()
```

Comment:
>MissingCodesInDisableNextLineDiagnosticsDirective_ShouldBeRecognizedWithError [](http://example.com/codeflow?start=20&length=77)

With lexer changes, you should also update some of the existing baseline tests because we have a bunch of round tripping tests that use those data sets.

Reply:
Added some baseline files as part of below two commits:
https://github.com/Azure/bicep/commit/3a23303bcb1c6f16a825cf431d768dd721985cb5
https://github.com/Azure/bicep/commit/c8a394a0db17a9407491c80f7c2f9f7adcf0fd66